### DISCUSSION TITLE
Drag Element Directly to Structure Panel

### DISCUSSION THREAD

**Discussion by Prof3ssorPants** [elementor/elementor#24455] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Dropping elements in the correct location/order without having to find the "sweet spot" within the editor.

### Describe the solution you'd like

I'd like to be able to drag elements like containers directly to the structure panel. This would make it much faster to set up and organize a page rather than having to drag to the editor, and then drag within the structure panel to re-arrange it correctly.

### Describe alternatives you've considered

The only alternative I can think of is just to drag and get lucky.

**Comment by perks98** [comment]
I completely agree. Since containers were released dropping to the right place isn't as great. I know they don't like hearing this but Bricks Builder has this feature and works well in terms of usage.
